company-profiles/brikl.md

```diff
@@ -0,0 +1,33 @@
+# Brikl
+
+## Company blurb
+
+Brikl is the leading eCommerce MicroStore technology partner for custom and promotional businesses worldwide. Our market-leading features include 15-minute store setup, bulk embellishment, and supplier integrations, vastly reducing the effort, cost, and chaos involved in the selling and customization process. We help build profitable businesses, increase employee happiness and offer internal efficiencies that positively impact the planet, enabling companies to create better across their organizations.
+
+## Company size
+
+100-150 Employees
+
+## Remote status
+
+5 days work week with flexible hours and WFH policy.
+
+## Region
+
+North America, Asia
+
+## Company technologies
+
+Brikl is the leading eCommerce MicroStore technology partner for custom and promotional businesses worldwide.
```

Comment:
One quick change I noticed, this section is usually where the technologies that the company uses goes, to make it eaisier for applicants to know whether they have the skills to apply for. 

Could you add some of the tech that the company uses, i.e Python, Node.js etc

Reply:
gotcha, and fixed